ValueSet/ValueSet-Genomics-types-of-reanalysistest.json

```diff
@@ -0,0 +1,32 @@
+{
+  "resourceType": "ValueSet",
+  "id": "genomics-reasonfortesting",
+  "url": "https://fhir.nhs.uk/ValueSet/genomics-types-of-reanalysistest",
+  "version": "0.1.0",
+  "name": "TypesofReAnalysisTestGenomics",
+  "title": "Types of Re-Analysis Test Genomics",
+  "status": "draft",
+  "date": "2024-07-09T00:00:00.000Z",
+  "publisher": "NHS England",
+  "contact": [
+    {
+      "name": "NHS England",
+      "telecom": [
+        {
+          "system": "email",
+          "value": "interoperabilityteam@nhs.net"
+        }
+      ]
+    }
+  ],
+  "description": "A ValueSet use to identify the types of re-analysis test request in genomics.",
```

Comment:
Could you add to the description where this is intended to be used? e.g. ServiceRequest.orderDetail?